common/static/common/js/components/views/search_field.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * A search field that works in concert with a paginated collection. When the user
+ * performs a search, the collection's search string will be updated and then the
+ * collection will be refreshed to show the first page of results.
+ */
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'underscore', 'text!common/templates/components/search-field.underscore'],
+        function (Backbone, _, searchFieldTemplate) {
+            return Backbone.View.extend({
+
+                events: {
+                    'submit .search-form': 'performSearch',
+                    'blur .search-form': 'performSearch',
+                    'click .action-clear': 'clearSearch'
+                },
+
+                initialize: function(options) {
+                    Backbone.View.prototype.initialize.call(this, options);
```

Comment:
I just don't see the need for the extra line noise and method call. I can't imagine Backbone would change `initialize` in a future version; that's what `constructor` is for.